spec/classes/foreman_proxy__plugin__openscap_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'spec_helper'
+
+describe 'foreman_proxy::plugin::openscap' do
+  on_supported_os.each do |os, facts|
+    context "on #{os}" do
+      context 'openscap plugin is enabled' do
+        let :params do
+          {
+            :enabled => true
+          }
+        end
+
+        it 'should call the plugin' do
+          should contain_foreman_proxy__plugin('openscap')
+        end
+
+        it 'should install configuration file' do
+          should contain_foreman_proxy__settings_file('openscap')
+          should contain_file('/etc/foreman-proxy/settings.d/openscap.yml').with({:content => /:enabled: https/})
```

Comment:
We have other tests where we test the entire file and I think that we should do the same here.